src/orchard/misc.clj

```diff
@@ -92,7 +92,8 @@
   "Parse a Java version string according to JEP 223 and return the appropriate version."
   [java-ver]
   (try
-    (let [[major minor _] (str/split java-ver #"\.")
+    (let [[no-opt _] (str/split java-ver #"-")
```

Comment:
I'd probably add some comment here explaining the need for this, as it's exactly obvious.